tsdb/tombstones/tombstones.go

```diff
@@ -317,12 +317,10 @@ func (itvs Intervals) Add(n Interval) Intervals {
 		}
 
 		if n.Mint < r.Mint {
-			newRange := make(Intervals, i, len(itvs[:i])+1)
-			copy(newRange, itvs[:i])
-			newRange = append(newRange, n)
-			newRange = append(newRange, itvs[i:]...)
-
-			return newRange
+			itvs = append(itvs, Interval{})
```

Comment:
This code is not equivalent.

Reply:
What do you mean for not equivalent? Does this change the logic? Or you mean it is unsafe?

Reply:
I'm not sure it's doing the same thing.